packages/frontend/src/components/DashboardTiles/TileBase/index.tsx

```diff
@@ -110,50 +112,55 @@ const TileBase = <T extends Dashboard['tiles'][number]>({
                         : {}
                 }
             >
-                <Tooltip
-                    disabled={!description || !!titleLeftIcon}
-                    label={description}
-                    multiline
-                    position="top-start"
-                    withinPortal
-                    maw={400}
-                >
-                    <TitleWrapper $hovered={titleHovered}>
-                        <Group spacing="xs">
-                            {titleLeftIcon}
+                {minimal ? (
+                    <Text fw={600} size="md">
+                        {title}
```

Comment:
need to check as well if `hideTitle` is truthy 👍

Reply:
good catch, updated